src/Etterna/Actor/Base/Actor.cpp

```diff
@@ -900,7 +900,7 @@ Actor::UpdateInternal(float delta_time)
 			break;
 		case CLOCK_TIMER_GLOBAL:
 			generic_global_timer_update(
-			  static_cast<float>(RageTimer::GetUsecsSinceStart()),
+			  RageTimer::GetTimeSinceStart(),
```

Comment:
This was a bug right. It looks like a bug, It used to be seconds before it somehow become microseconds